Yoast/Yoast-SEO-for-TYPO3
# [BUGFIX] Added html_entity_decode to title and description within snippet preview

## Summary

This PR can be summarized in the following changelog entry:

* Html entities within title and description are now being decoded, to prevent displaying html within the snippet preview

## Relevant technical choices:

* Used the simple `html_entity_decode` function

## Test instructions

This PR can be tested by following these steps:

* Add some html entities to your title and description and see the encoded characters within the snippet preview
* Pull this branch
* Reload the page in the backend and check if it's displayed correctly now

## Quality assurance


Fixes #307
